test/e2e/common/node/container_probe.go

```diff
@@ -209,6 +210,22 @@ var _ = SIGDescribe("Probing container", func() {
 		RunLivenessTest(f, pod, 0, defaultObservationTimeout)
 	})
 
+	/*
+		Release: v1.22
+		Testname: Pod liveness probe, using grpc call, failure
+		Description: A Pod is created with liveness probe on grpc service. Liveness probe on this endpoint will not fail. When liveness probe does not fail then the restart count MUST remain zero.
+	*/
+	framework.ConformanceIt("should *not* be restarted with a GRPC liveness probe [NodeConformance]", func() {
```

Comment:
too early for conformance. The feature is in alpha

Reply:
I am not familar with e2e test, what should i todo in e2e test for this feature.